stable_horde/horde.py

```diff
@@ -122,14 +122,13 @@ def detect_current_model(self):
         if checkpoint_info is None:
             return f"Model checkpoint {model_checkpoint} not found"
 
-        local_hash = get_sha256sum(checkpoint_info.filename)
         for model in self.supported_models:
             try:
                 remote_hash = model["config"]["files"][0]["sha256sum"]
             except KeyError:
                 continue
 
-            if local_hash == remote_hash:
+            if shared.opts.sd_checkpoint_hash == remote_hash:
```

Comment:
I can see now stable horde has every model hashed in md5 and sha256, and sd-webui is using sha256 hash as well, so we could use sha256 hash everywhere, if I understand it correctly?
I think the `get_md5sum` or `get_sha256sum` can be removed since sd-webui has exposed the hash calculation function

Reply:
You are correct. We don't need our own functions anymore. I completly missed that sorry. Will fix tomorrow!